compiler-plugin/src/main/kotlin/com/google/devtools/ksp/symbol/impl/utils.kt

```diff
@@ -235,41 +235,41 @@ internal fun ModuleClassResolver.resolveContainingClass(psiMethod: PsiMethod): C
 }
 
 internal fun getInstanceForCurrentRound(node: KSNode): KSNode? {
-    when (node.origin) {
+    return when (node.origin) {
         Origin.KOTLIN_LIB, Origin.JAVA_LIB -> return null
```

Comment:
Could you remove the `return` in `return null`?

Reply:
done.

thank you for telling me